validator/client/validator.go

```diff
@@ -166,6 +166,7 @@ func (v *validator) WaitForSync(ctx context.Context) error {
 	for {
 		select {
 		// Poll every half slot
+		// Why are we polling every half slot here?
 		case <-time.After(time.Duration(params.BeaconConfig().SlotsPerEpoch/2) * time.Second):
```

Comment:
I think this was arbitrary, but maybe we should create a constant at the top or something that's called syncPollingInterval

Reply:
yeah that sounds good